keyboards/frobiac/blackflat/rules.mk

```diff
@@ -0,0 +1,6 @@
+PS2_MOUSE_ENABLE = yes
+PS2_ENABLE = yes
+PS2_DRIVER = busywait
+
+# required if keymap is not named the same as userspace to use
+USER_NAME := frobiac
```

Comment:
Sorry, by `default` keymap I mean a simple keymap with bare minimum functionality that doesn't rely on anything outside of the keymap itself (like a userspace). This is more what it should look like
```
keymaps
├── default
│   └── keymap.c
└── frobiac
    └── keymap.c
```
```suggestion
```

Reply:
Sorry, my bad - should have been clear from your initial comment, but thanks for the clarification.